### ISSUE TITLE
Request: add option to configure a different remote user for transfer_scp

### ISSUE THREAD

**Issue by jrhartog** [usgs/shakemap#1119]
We would like to transfer the ShakeMap to the account of a different user on a remote server using transfer_scp. Currently, transfer_scp seems to assume the remote user is the same as the user running the transfer_scp program.

```
(shakemap) [shake@products02 shakemap]$ python setup.py version. (from forked version pnsn/shakemap so I can hack these things in for now, all unit tests pass).

running version
keywords are unexpanded, not using
got version from VCS {'version': '4.0.2+432.gc696f4a.dirty', 'full-revisionid': 'c696f4a1c1def57f132908cded299585e5da3700', 'dirty': True, 'error': None, 'date': '2021-01-29T14:09:31-0800'}
Version: 4.0.2+432.gc696f4a.dirty
 full-revisionid: c696f4a1c1def57f132908cded299585e5da3700
 dirty: True
 date: 2021-01-29T14:09:31-0800
(shakemap) [shake@products02 shakemap]$ uname -v
#1 SMP Tue Aug 25 17:23:54 UTC 2020
(shakemap) [shake@products02 shakemap]$ conda --version
conda 4.9.2
```
---
If you are reporting a problem with running the code, please report the outputs from the following commands after cloning this reponsitory and running `install.sh`:
    - `python setup.py version`
    - `uname -v`
    - `conda --version`

**Comment by cbworden** [comment]
Hi Renate. This is something we are working on and should have available as soon as we fix a few other things. Impactutils has been modified to accept an account name other than "shake", and we just need to modify transfer.conf, transferspec.conf, and transfer_scp.py to include a user name. If you have that code, you could submit a pull request, or you can just wait a bit and I'll get to it.

**Comment by jrhartog** [comment]
Hi Bruce, I created a pull request. As long as the user has the latest version of impactutils installed the new option should work.

**Comment by cbworden** [comment]
Thanks, that's a big help. I merged the PR, so it should be available in the repo now.
Thanks again,
B
Closed by #1122
